mm/src/code/z_parameter.c

```diff
@@ -21,7 +21,9 @@
 #include "2s2h/BenGui/HudEditor.h"
 #include "2s2h/BenGui/CosmeticEditor.h"
 #include "2s2h_assets.h"
+#include "gfxbridge.h"
```

Comment:
Use full path like the other bridges
```suggestion
#include "public/bridge/gfxbridge.h"
```